portfolio/src/main/webapp/script.js

```diff
@@ -60,8 +60,29 @@ function getRandomElementfromTopic() {
   setElementText(elem, 'elem-container');
 }
 
-async function getGreetingfromServer() {
+/**
+ * Creates an <li> element for comments-container containing text. 
+ */
+function createCommentsListElement(text, i) {
```

Comment:
nit: from an API design perspective, it might be better if this param was the CSS class name instead. Otherwise, as a user of this API, I would have to know what `i` is and what the effect of it is.